src/main/kotlin/com/replaymod/gradle/preprocess/PreprocessTask.kt

```diff
@@ -238,6 +241,8 @@ class CommentPreprocessor(private val vars: Map<String, Int>) {
     fun convertSource(kws: Keywords, lines: List<String>, remapped: List<Pair<String, List<String>>>, fileName: String): List<String> {
         val ifStack = mutableListOf<Boolean>()
         val indentStack = mutableListOf<Int>()
+        var trueConditionFound = false
+        var elseFound = false
```

Comment:
This will have to be a stack similar to `ifStack`, otherwise nested ifs can incorrectly throw.